src/components/SecondaryNav.tsx

```diff
@@ -36,7 +36,7 @@ const SecondaryNav = ({ activeLink, user }: SecondaryNavProps): JSX.Element => {
     <div>
       <div className="bg-darkestGrey py-14 md:py-16">
         <nav className="container">
-          <div className="flex flex-col space-y-2 sm:flex-row text-xl font-righteous text-accent font-bold justify-center items-baseline cursor-pointer">
+          <div className="flex space-y-2 flex-row text-xl font-righteous text-accent font-bold justify-center items-baseline cursor-pointer">
```

Comment:
> @KrisMannino Looks better! Can you set a smaller text on the smaller break-point, so that the nav links also fit well even on a smaller screen? 😀

May I suggest this:

```suggestion
          <div className="flex space-y-2 flex-row text-sm sm:text-xl font-righteous text-accent font-bold justify-center items-baseline cursor-pointer">
```